Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -763,6 +763,9 @@ class Outputs:
     cluster_roles = ["Attribute", "Class variable", "Meta variable"]
     basic_annotations = ["None", "Enumeration"]
 
+    class Error(widget.OWWidget.Error):
+        not_finite_distances = Msg("Not all distances are finite")
```

Comment:
You're right, we have to decide. @BlazZupan, @astaric, @lanzagar, @ajdapretnar? I don't have a clear preference.